doc/releasenotes/16_0_0_summary.md

```diff
@@ -56,6 +56,10 @@ The flag is available in these components:
 
 ### Breaking Changes
 
+#### Default MySQL version on Docker
+
+The default MySQL version use by our `vitess/lite:latest` image is going from `5.7` to `8.0`. Additionally, the default version of the `vitess/lite:mysql80` image goes from `8.0.23` to `8.0.31`.
```

Comment:
`MySQL version used` and `the default patch version of the`... we can make touch-ups like that as part of the release process though.

Reply:
Follow up PR: https://github.com/vitessio/vitess/pull/12222